scala-runner

```diff
@@ -27,7 +27,7 @@ addCoursier () {
   coursier_args+=("$1")
 }
 getScalaNightly() {
-  . <(curl https://raw.githubusercontent.com/scala/community-builds/"$1"/nightly.properties 2>/dev/null)
+  local nightly=$(curl https://raw.githubusercontent.com/scala/community-builds/"$1"/nightly.properties 2>/dev/null | sed -n 's/nightly=\(.*\)/\1/p')
```

Comment:
https://stackoverflow.com/questions/8225822/how-to-make-sed-remove-lines-not-matched-by-a-substitution